epub33/core/index.html

```diff
@@ -5478,7 +5442,64 @@ <h4>Path and File Names</h4>
+							</ol>	
+						</li>
+						<li>
+							Return the <a data-cite="infra#string-concatenate">concatenation</a> of <var>path</var> using the  <code>U+002F (/)</code> character.
+						</li>
+					</ol>
+				</section>
+
+				<section id="sec-container-iri">
+					<h4>URLs in the OCF Abstract Container</h4>
+
+					<p>The <a>container root URL</a> is the <a data-cite="url#concept-url">URL</a> [[URL]] of the
+						<a>Root Directory</a>. It is implementation specific, but EPUB Creators MUST assume it has the following properties:</p>
+
+					<ul>
+						<li>The result of <a data-cite="url#concept-url-parser">parsing</a> "<code>/</code>" with the <a>container root URL</a> as <a data-cite="url#concept-base-url"><var>base</var></a> is the <a>container root URL</a>.</li>
+						<li>The result of <a data-cite="url#concept-url-parser">parsing</a> "<code>..</code>" with the <a>container root URL</a> as <a data-cite="url#concept-base-url"><var>base</var></a> is the <a>container root URL</a>.</li>	
+					</ul>
+
+					<p>The <a>content URL</a> of a file or directory in the <a>OCF Abstract Container</a> is the result of <a data-cite="url#concept-url-parser">parsing</a> 
+						the file's <a>File Path</a> with the <a>container root URL</a> as <a data-cite="url#concept-base-url"><var>base</var></a>.</p>
+
+					<div class="note">
+						<p>
+							<a data-cite="url#concept-url-parser">Parsing</a> may replace some charaters in the File Path by their <a data-cite="url#percent-encode">percent encoded</a> alternative. For example, <code>A/B/C/file&nbsp;name.xhtml</code> becomes <code>A/B/C/file%20name.xhtml</code>. 
+						</p>
+					</div>
+
+					<p>
+						In the <a>OCF Abstract Container</a>, when a file uses a URL string to reference another file in the container, the string MUST be a 
+						<a data-cite="url#path-relative-scheme-less-url-string">path-relative-scheme-less-URL string</a>, optionally followed by <code>U+0023 (#)</code> and a <a data-cite="url#url-fragment-string">URL-fragment string</a>.
+					</p>
+
+					<aside class="example">
+						<p>The following example shows how to reference, from an [[HTML]] <code>img</code> element, a
+							file named <code>image1.jpg</code> in the same directory as an <a>XHTML Content
+							Document</a>.</p>
+						<pre>&lt;img src="image1.jpg" alt="…" /&gt;</pre>
+					</aside>
 
+					<p class="note">
+						The properties of the <a>container root URL</a> are such that whatever the amount of <a data-cite="url/#double-dot-path-segment">double-dot path segments</a> in a URL string (for example, <code>../../../secret</code>), it will be parsed to a content URL (and not "leak" outside the container). However, for better interoperability with non-conforming or legacy Reading Systems, EPUB Creators should avoid to use more <a data-cite="url/#double-dot-path-segment">double-dot path segments</a> than needed to reach the target container file.
```

Comment:
Nit: replace "to use" with "using"

Side question: if I write 

s/to use/using/

Do people know what I mean?

Reply:
Done.

(On the side question: I do...)

Reply:
Done.

(On the side question: I do...)